tensorboard/plugins/interactive_inference/witwidget/notebook/jupyter/js/lib/wit.js

```diff
@@ -40,9 +41,14 @@ var WITView = widgets.DOMWidgetView.extend({
     const height = parseInt(
       this.model.attributes.layout.attributes.height, 10) - 20;
     const iframe = document.createElement('iframe');
-    const templateLocation =
-      window.__webpack_public_path__ + 'wit_jupyter.html';
-    const src = `<link rel="import" href="${templateLocation}">
+
+    // Adjust WIT html location if running in a jupyter notebook
+    // and not in jupyterlab.
+    if (document.querySelector('body').getAttribute('data-base-url') != null) {
+      witHtmlLocation = window.__nbextension_path__ + 'wit_jupyter.html';
+    }
+
+    const src = `<link rel="import" href="${witHtmlLocation}">
```

Comment:
Please note that this will break in Chrome M75, which is [scheduled to
be released… uh, today.][calendar] Maybe test with Chrome beta/canary?

See [my previous comment for migration instructions][migration].

[calendar]: https://chromiumdash.appspot.com/schedule
[migration]: https://github.com/tensorflow/tensorboard/pull/2216#discussion_r282723286

Reply:
thanks. will handle this in separate PR ASAP

Reply:
Sounds good; thanks!

Reply:
I tested WIT in colab in chrome canary (which is running 77.0.3815.0) and it works as currently implemented.